src/game_battlealgorithm.cpp

```diff
@@ -348,17 +343,16 @@ std::string Game_BattleAlgorithm::AlgorithmBase::GetParameterChangeMessage(bool
 	const std::string& message = is_positive ?
 		Data::terms.parameter_increase :
 		Data::terms.parameter_decrease;
-	std::stringstream ss;
 
 	if (Player::IsRPG2kE()) {
-		ss << value;
 		return Utils::ReplacePlaceholders(
 			message,
 			{'S', 'V', 'U'},
-			{GetTarget()->GetName(), ss.str(), points}
+			{GetTarget()->GetName(), std::to_string(value), points}
```

Comment:
We had problems with missing std::to_string on some platforms years ago (we have Utils::ToString as an alternative) but I will recheck if this is still problematic.